examples/erc20DividendSandbox.ts

```diff
@@ -0,0 +1,161 @@
+import { BigNumber } from '@0x/utils';
+import { RedundantSubprovider, RPCSubprovider, Web3ProviderEngine } from '@0x/subproviders';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import { EtherDividendCheckpointEvents } from '@polymathnetwork/abi-wrappers/lib/src';
+import {ApiConstructorParams, PolymathAPI} from '../src/PolymathAPI';
+import {valueToWei, weiToValue} from '../src/utils/convert';
+import {ModuleName, ModuleType} from '../src';
+
+// This file acts as a valid sandbox for adding a etherDividend  module on an unlocked node (like ganache)
+
+window.addEventListener('load', async () => {
+  // Setup the redundant provider
+  const providerEngine = new Web3ProviderEngine();
+  providerEngine.addProvider(new RedundantSubprovider([new RPCSubprovider('http://127.0.0.1:8545')]));
+  providerEngine.start();
+  const params: ApiConstructorParams = {
+    provider: providerEngine,
+    polymathRegistryAddress: '<Deployed Polymath Registry address>',
+  };
+
+  // Instantiate the API
+  const polymathAPI = new PolymathAPI(params);
+
+  // Get some poly tokens in your account and the security token
+  const web3Wrapper = new Web3Wrapper(params.provider, {
+    gasPrice: params.defaultGasPrice,
+  });
+  const address = await web3Wrapper.getAvailableAddressesAsync();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
+  await polymathAPI.getPolyTokens({
```

Comment:
Use `transfer` instead

Reply:
Based on this, does it mean that I should only getPolyTokens for polymathAPI.getAccount() ? In that any time the module, security token(registry) needs tokens we need to transfer them from our local account?

Reply:
Yeah, it means exactly that. Tokens should be requested from the faucet only for the account that plans to later transfer them to another contract.